docs/api/baseplate/frameworks/queue_consumer/kombu.rst

```diff
@@ -0,0 +1,53 @@
+``baseplate.frameworks.queue_consumer.kombu``
+=============================================
+
+:doc:`Kombu <kombu:index>` is a library for interacting with queue brokers.
+
+This module provides a :py:class:`~baseplate.server.queue_consumer.QueueConsumerFactory`
+that allows you to run a :py:class:`~baseplate.server.queue_consumer.QueueConsumerServer`
+using Kombu that also integrates Baseplate's facilities into Kombu's message lifecycle.
+
+An abbreviated example of it in use::
+
+    def process_links(context, body, message):
+        print(f"processing {body}")
+
+    def make_consumer_factory(app_config):
+        baseplate = Baseplate()
+        exchange=Exchange("reddit_exchange", "direct"),
```

Comment:
these bits look like they're written as params to a function but they're top-level statements.

maybe turn this into a doctest so at least we know it's valid syntax even if we don't go all the way with it?

Reply:
> these bits look like they're written as params to a function but they're top-level statements.

Sorry about that, I fixed that in a later commit (copy/paste oversight).